xbmc/settings/MediaSettings.cpp

```diff
@@ -94,7 +94,7 @@ bool CMediaSettings::Load(const TiXmlNode *settings)
       scalingMethod = (int)VS_SCALINGMETHOD_LINEAR;
     m_defaultVideoSettings.m_ScalingMethod = (ESCALINGMETHOD)scalingMethod;
 
-    XMLUtils::GetInt(pElement, "viewmode", m_defaultVideoSettings.m_ViewMode, ViewModeNormal, ViewModeCustom);
+    XMLUtils::GetInt(pElement, "viewmode", m_defaultVideoSettings.m_ViewMode, ViewModeNormal, ViewModeZoom110Width);
```

Comment:
I struggled with this particular bit of code for a little while. I didn't realize it affected the GUI, though. I thought it just loaded the video settings. I didn't want to change the numbering of any of the existing view modes as that could mess up existing settings. But I also saw that the last view mode was intentionally excluded from this. I honestly don't know enough about the code to make a good decision on this one. If you or anyone else has any suggestions, let me know.